### PR TITLE
Feature/lga 1866 delete user

### PR DESCRIPTION
**Pull request by psweeting1**
- New steps for logging into newly created operator in CHS
- New steps checking dashboard page on CHS
- New steps for checking delete user page (Are you sure page?)
- New steps for delete user
- Addressing older steps and cleaning up visual issues.

### CODE REVIEW

File: behave/features/foxadmin_users.feature [hunk 1]
```diff
@@ -8,22 +8,34 @@ Background: Log in to the Fox admin as a user within the super users group
...
-And I select 'Is active’
+And I select 'Is active'
 And I choose to "save"
 Then the new operator user is created
 And I am taken to the list of operators page
+#Child 3
 And I select the link "Log out"
-And that I am logged in as "FOX_ADMIN_NEW_USER"
\ No newline at end of file
+And that I am logged in as "FOX_ADMIN_NEW_USER"
```
Comment (said-moj) [inline behave/features/foxadmin_users.feature:31]: This is a bit confusing as you are logging into the CHS but as a new fox admin user
What do you think of renaming that parameter to something like 'NEWLY_CREATED_OPERATOR' or something like that?

File: behave/features/steps/foxadmin.py [hunk 2]
```diff
@@ -186,5 +187,57 @@ def step_impl(context):
...
+        pass
+
+
+@step(u'I select the \'Yes, I\'m sure\'')
+def step_impl(context):
+    yes_btn = "//input[@type='submit']"
+    context.helperfunc.click_button(By.XPATH, yes_btn)
+
+
+@step(u'I am on the call centre dashboard')
```
Comment (said-moj) [inline behave/features/steps/foxadmin.py:237]: There is already same/similar step defined here https://github.com/ministryofjustice/cla-end-to-end-tests/blob/main/behave/features/steps/test_steps.py#L8

File: behave/features/foxadmin_users.feature (hunk 1 above)
Comment (psweeting1) [reply behave/features/foxadmin_users.feature]: Yeah I can see that. I've made the changes

File: behave/features/steps/foxadmin.py (hunk 2 above)
Comment (psweeting1) [reply behave/features/steps/foxadmin.py]: Good thing there is already step, I just couldn't find that when I was looking.

**Review by said-moj — APPROVED** [review]
👍 LGTM

**Pull request merged**